docs/src/features.md

```diff
@@ -142,6 +142,21 @@ This package presents Code Lenses as a greenish clickable annotation positioned
 
 Sublime Text has no concept of Code Lenses.
 
+## Inlay Hints
+
+Inlay hints are short textual annotations that show parameter names, type hints.
+
+![inlay-hints](./images/inlay-hints.png)
+
+To enable inlay hints, modify the following settings through `Preferences: LSP Settings`:
+```js
+{
+    "show_inlay_hints": true
+}
+```
```

Comment:
I suppose we can just mention which option to enable. Including a whole code block with just that option sounds a bit excessive.

```suggestion
To enable inlay hints, enable the `show_inlay_hints` setting through `Preferences: LSP Settings`.
```